autogalaxy/operate/image.py

```diff
@@ -15,53 +17,53 @@ class OperateImage:
     The methods in `OperateImage` are inherited by light objects to provide a concise API.
     """
 
-    def image_2d_from(self, grid: Union[aa.Grid2D, aa.Grid2DIterate]) -> aa.Array2D:
+    def image_2d_from(
+        self,
+        grid: Union[aa.Grid2D, aa.Grid2DIterate],

```

Comment:
Grid2DIterate inherits from Grid2D so there is no point in the union - just use the type aa.Grid2D